pkg/controller/vsphere/util.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 const (
-	globalInfrastuctureName  = "cluster"
-	openshiftConfigNamespace = "openshift-config"
+	globalInfrastuctureName         = "cluster"
+	OpenshiftConfigNamespace        = "openshift-config"
+	openshiftConfigNamespaceForTest = "openshift-config-test"
```

Comment:
This constant should probably be in one of the test files, is there a suite file or somewhere we could put it?

Alternatively, create a new namespace for each test using the `generateName` feature. We do that a lot in our test suites

Reply:
moved this const `openshiftConfigNamespaceForTest` to `util_test.go` with other const for tests